### PR TITLE
DDBTEAM-423: Filter out small images

### PR DESCRIPTION
**Pull request by cableman**
https://jira.itkdev.dk/browse/DDBTEAM-423

### CODE REVIEW

File: src/Api/DataProvider/AbstractElasticSearchDataProvider.php [hunk 1]
```diff
@@ -58,8 +60,10 @@ abstract class AbstractElasticSearchDataProvider
      *   Service for registering no hits
      * @param loggerInterface $logger
      *   Standard logger
+     * @param ParameterBagInterface $params
+     *   Access to environment variables
      */
-    public function __construct(Type $index, RequestStack $requestStack, StatsLoggingService $statsLoggingService, MetricsService $metricsService, EventDispatcherInterface $dispatcher, IdentifierFactory $factory, NoHitService $noHitService, LoggerInterface $logger)
+    public function __construct(Type $index, RequestStack $requestStack, StatsLoggingService $statsLoggingService, MetricsService $metricsService, EventDispatcherInterface $dispatcher, IdentifierFactory $factory, NoHitService $noHitService, LoggerInterface $logger, ParameterBagInterface $params)
```
Comment (tuj) [inline src/Api/DataProvider/AbstractElasticSearchDataProvider.php]: It is better to inject the variable than the parameter bag.

File: src/Api/DataProvider/AbstractElasticSearchDataProvider.php [hunk 2]
```diff
@@ -107,21 +114,40 @@ protected function registerSearchNoHits(string $type, array $identifiers): void
...
-            $boolQuery->addShould($identifierFieldTermQuery);
+            $innerQuery->addShould($identifierFieldTermQuery);
         }
 
+        $outerBoolQuery = new Query\BoolQuery();
+        $outerBoolQuery->addMust($innerQuery);
+
+        $range = new Query\Range();
+        $range->addField('height', [
+            'gte' => $this->params->get('elastic.min.image.size'),
```
Comment (tuj) [inline src/Api/DataProvider/AbstractElasticSearchDataProvider.php]: What if this has not been set?
Better to initialize the variable in the constructor in my opinion.

File: src/Api/DataProvider/AbstractElasticSearchDataProvider.php (hunk 1 above)
Comment (cableman) [reply src/Api/DataProvider/AbstractElasticSearchDataProvider.php]: I have just followed the pattern used in the rest of cover-service and cover-service-importers!

File: src/Api/DataProvider/AbstractElasticSearchDataProvider.php (hunk 2 above)
Comment (cableman) [reply src/Api/DataProvider/AbstractElasticSearchDataProvider.php]: Fixed

**Pull request merged**